### DISCUSSION TITLE
有没有机会换一个语言(就是全平台啦)

### DISCUSSION THREAD

**Discussion by SaoHuangDaDui233** [Meloong-Git/PCL#83] [提问] [answered]
玩mc用mac和linux虽然没有windoes那么多，但人数也不少啊

**Comment by Light-Beacon** [comment]
这...完全无异于是重写一个启动器，估计lm不会做的

**Comment by LTCatt** [comment, marked as answer]
没机会.jpg

**Comment by SorrowRain** [comment]
PCL2可以在wine最新版上跑起来了=。=，但是仍然存在兼容性问题，可以试试

**Reply by zkitefly** [reply]
> PCL2可以在wine最新版上跑起来了=。=，但是仍然存在兼容性问题，可以试试

可以吗，那你打包，公开下QWQ，我也想用（
